lib/practice_exercises.rb

```diff
@@ -1,13 +1,31 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 def is_palindrome(string)
```

Comment:
👍 Correct time complexity and space complexity due to the array you create with `string.gsub(/[^a-z^0-9]/i, '').downcase.split("")`